app/utils.py

```diff
@@ -8,9 +8,36 @@
 from app.generation.common.vector import Vector2
 from app.version import Version
 
+# osu api v2
+from ossapi import Ossapi, UserLookupKey
+from oauthlib.oauth2.rfc6749.errors import InvalidClientError
+
+
+API_KEY_FILE: Path = Path.cwd() / "apikey.txt"
+class api:
+    def __init__(self):
+        self.api: Ossapi | None = self.get_api()
+
+    @classmethod
+    def get_api(self) -> Ossapi | None:
+        try:
+            CLIENT_ID, CLIENT_SECRET = API_KEY_FILE.read_text().split("\n")[:2] # split the file by new line
```

Comment:
you can move that part of the code from get_user_avatar here (and also probably rename this function to _get_api because we probably dont want this to get called externally)